Userland/Libraries/LibJS/Runtime/Completion.h

```diff
@@ -150,9 +147,8 @@ class [[nodiscard]] Completion {
         return m_type == Type::Empty;
     }
 
-    Type m_type { Type::Normal };           // [[Type]]
-    Optional<Value> m_value;                // [[Value]]
-    Optional<DeprecatedFlyString> m_target; // [[Target]]
+    Type m_type { Type::Normal }; // [[Type]]
+    Optional<Value> m_value;      // [[Value]]
```

Comment:
Would it maybe be useful here to leave a comment somewhere about the fact it is not used in the constructor incase someone working on libjs is adding something to spec and needs to figure out about why it isn't required.

From digging into the spec it seems unused because `[[Target]]` is meant to refer to a label, but that's all internally handled in bytecode and not exposed to anything in runtime (if I am understanding everything correctly here?)

Reply:
No harm in adding a comment about that! You are indeed right that it's control flow information that the bytecode already handles. This was useful back in the AST interpreter times. :)